lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.pm

```diff
@@ -0,0 +1,375 @@
+    $stats{'false_positive_exact'}   = $cmd->stat_false_positive_exact();
+    $stats{'false_positive_partial'} = $cmd->stat_false_positive_partial();
+    $stats{'true_negatives'}         = $cmd->stat_true_negatives();
+    $stats{'exact_specificity'}      = $cmd->stat_exact_specificity();
+    $stats{'partial_specificity'}    = $cmd->stat_partial_specificity();
+    $stats{'exact_ppv'}              = $cmd->stat_exact_ppv();
+    $stats{'partial_ppv'}            = $cmd->stat_partial_ppv();
+
+    $stats{'vcf_lines_overlapping_tn'} =
+      $cmd->stat_vcf_lines_overlapping_tn();
+
+    $stats{'lines_specificity_in_tn_only'} =
+      $cmd->stat_lines_specificity_in_tn_only();
+
+    return \%stats;
+}
+
+sub stat_types {
+    my $self = shift;
+
+    my @types = (
+        'true_positive_found_exact', 
+        'total_true_positive_exact', 
+        'sensitivity_exact', 
+        'true_positive_found_partial', 
+        'total_true_positive_partial', 
+        'sensitivity_partial', 
+        'false_positive_exact', 
+        'false_positive_partial', 
+        'true_negatives', 
+        'exact_specificity', 
+        'partial_specificity', 
+        'exact_ppv', 
+        'partial_ppv', 
+        'vcf_lines_overlapping_tn', 
+        'lines_specificity_in_tn_only', 
+    );
+
+    return @types;
+}
+
+sub create_subdirectory_workspace {
+    my ($self, $inputs) = @_;
+    my $base = $self->fetch_output_directory;
+    my ($name, $id, $variant_type) = @{$inputs}{'name', 'id', 'variant_type'};
+
+    my $wkspace = join('_', $name, $id);
+    my $subdir = $base->subdir($wkspace)->subdir($variant_type);
+    $subdir->mkpath or die $self->error_message(
+        "Couldn't make workspace path '$subdir' on the file system!"
+    );
+
+    return $subdir;
+}
+
+sub validate_type {
+    my ($self, $type) = @_;
+    if ($type ne 'snvs' && $type ne 'indels') {
+        die $self->error_message(
+            "Invalid type: '$type'. Only 'snvs' and 'indels' are supported "
+            . "as valid types."
+        );
+    }
+    return 1;
+}
+
+sub fetch_config_file {
+    my $self = shift;
+    my $config = Path::Class::File->new($self->config_file());
+    unless (-e $config) {
+        die $self->error_message("Couldn't find $config on the file system!");
+    }
+
+    return $config;
+}
+
+sub fetch_output_directory {
+    my $self = shift;
+    my $dir = Path::Class::Dir->new($self->output_directory());
+    unless (-d $dir) {
+        $self->status_message(
+            "'$dir' doesn't exist on file system."
+            . "Making path"
+        );
+
+        $dir->mkpath or die $self->error_message(
+            "Couldn't make directory $dir on the file system!"
+        );
+    }
+
+    return $dir;
+}
+
+sub help_brief {
+    return "Validate multiple VCF files to a specified Gold Standard Set";
+}
+
+sub help_detail {
+    my $doc = q{
+      This tool is a wrapper process to Genome::Model::Tools::Vcf::EvaluateVcf.
```

Comment:
It might make more sense for the documentation to refer to `gmt vcf evaluate-vcf` than its Perl class.